extension/scripts/Scraper.js

```diff
@@ -90,7 +90,9 @@ Scraper.prototype = {
     job.execute(this.browser, function (err, job) {
       if (err) {
         // jobs don't seem to return anything
-        return console.error('Error in job', err)
+        this.executionCallback(err)
+        console.error('Error in job', err)
```

Comment:
Should we first log and then execute the callback just in case something fails and we do not reach this line?

Reply:
Agreed